src/test/java/com/exasol/adapter/dialects/AbstractIntegrationTest.java

```diff
@@ -0,0 +1,93 @@
+package com.exasol.adapter.dialects;
+
+import static com.exasol.adapter.dialects.IntegrationTestConstants.*;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.sql.*;
+import java.util.List;
+import java.util.Properties;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+
+import com.exasol.bucketfs.Bucket;
+import com.exasol.bucketfs.BucketAccessException;
+
+public abstract class AbstractIntegrationTest {
```

Comment:
this class makes no sense anymore. Please remove its content to src/test/java/com/exasol/adapter/dialects/hive/HiveSqlDialectIT.java. Also change the `protected` modifiers by `private`